src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -426,12 +446,21 @@ class PrisonerIepLevelReviewService(
     return review
   }
 
-  private suspend fun publishDomainEvent(
+  private suspend fun publishReviewDomainEvent(
     iepDetail: IepDetail,
     eventType: IncentivesDomainEventType,
   ) {
     iepDetail.id?.let {
-      snsService.sendIepReviewEvent(iepDetail.id, iepDetail.prisonerNumber ?: "N/A", iepDetail.iepTime, eventType)
+      val description: String = when (eventType) {
+        IncentivesDomainEventType.IEP_REVIEW_INSERTED -> "An IEP review has been added"
+        IncentivesDomainEventType.IEP_REVIEW_UPDATED -> "An IEP review has been updated"
+        IncentivesDomainEventType.IEP_REVIEW_DELETED -> "An IEP review has been deleted"
+        else -> {
+          throw IllegalArgumentException("Tried to publish a review event with a non-review event type: $eventType")
```

Comment:
as there are no other variants of `IncentivesDomainEventType`, is this `else` block there to indicate an error in implementation?

Reply:
Pretty much, it's basically to highlight our code tried to publish a review domain event with an unexpected domain event type.